apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx

```diff
@@ -1,9 +1,149 @@
+import { Link } from 'react-router-dom';
 
 type Props = {
     result: PatientSearchResult;
 };
 
-const PatientSearchResultListItem = ({ result }: Props) => <>{result.patient}</>;
+const PatientSearchResultListItem = ({ result }: Props) => {
+    const { legalName, birthday, gender, shortId, phones, names, emails, addresses, identification } = result;
+    const noData = 'No Data';
+    const driversLicense = identification.find((id) => id.type === "Driver's license number");
+    const ssn = identification.find((id) => id.type === 'Social Security');
+    const first = legalName?.first ?? '';
+    const last = legalName?.last ?? '';
+    const firstName = first.charAt(0).toUpperCase() + first.slice(1).toLowerCase();
+    const lastName = last.charAt(0).toUpperCase() + last.slice(1).toLowerCase();
+
+    return (
+        <div className={styles.listItem}>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="legalName" className={styles.listItemLabel}>
+                        LEGAL NAME
+                    </label>
+                    <br />
+                    <Link
+                        id="legalName"
+                        className={`${styles.value}, ${styles.name}`}
+                        to={`patient-profile/${result.patient}/summary`}>
+                        {lastName && firstName ? `${lastName}, ${firstName}` : noData}
+                    </Link>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="dob" className={styles.listItemLabel}>
+                        DATE OF BIRTH
+                    </label>
+                    <span id="dob" className={styles.value}>
+                        {birthday ?? noData}
+                    </span>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="sex" className={styles.listItemLabel}>
+                        SEX
+                    </label>
+                    <span id="sex" className={styles.value}>
+                        {gender ?? noData}
+                    </span>
+                </div>
+                <div className={styles.listItemData}>
+                    <label htmlFor="patientId" className={styles.listItemLabel}>
+                        PATIENT ID
+                    </label>
+                    <span id="patientID" className={styles.value}>
+                        {shortId}
+                    </span>
+                </div>
+            </div>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="phone" className={styles.listItemLabel}>
+                        PHONE
+                    </label>
+                    <br />
+                    <span id="phone" className={styles.value}>
+                        {phones[0] ?? noData}
```

Comment:
question: are we displaying only one phone number on the list item?

Reply:
I'll follow up with design